provider/k8s.go

```diff
@@ -208,7 +209,7 @@ func k8sOfflineStoreFactory(config pc.SerializedConfig) (Provider, error) {
 
 	serializedBlob, err := k8.StoreConfig.Serialize()
 	if err != nil {
-		return nil, fmt.Errorf("could not serialize blob store config")
+		return nil, err
```

Comment:
do we need an error type here? if not ignore

Reply:
Nah its handled by another function